### DISCUSSION TITLE
Support for Incremental Policy Rollout

### DISCUSSION THREAD

**Discussion by zanebclark** [openmobilityfoundation/curb-data-specification#92] [Data Specification]
A use case: 
A city defines a policy that's reusable across multiple zones and the policy is "applied" to zones incrementally. That is, signage goes up at zone A on Monday and at zone B on Tuesday. A policy time span can define a start time for a specific policy, but this start time would apply across all zones. In this case the start time of the policy in zone A should differ from the start time of the policy in zone B. 

Is there support for this sort of incremental rollout of a policy in CDS? Internally, we have a join table that defines the relationship between a zone and a policy. We can use the created_at timestamp of this relationship to start enforcement of the policy after the time of the relationship's creation. Since CDS doesn't have an entity reflecting the relationship between zone and policy, I'm not sure where this information would go.

**Comment by jlarsonOmahaNE** [comment]
I believe that a curb zone does reference a policy based on the id of that policy. Would the time_spans field work for this in the policies?

**Reply by zanebclark** [reply]
@jlarsonOmahaNE , if you define a policy with a time span and then apply the policy to one zone on Monday and a second zone on Wednesday, the time span for the policy will remain the same across both zones.
